.github/workflows/integration-tests.yml

```
@@ -74,3 +74,23 @@ jobs:
       - run: pip install -ve .
       - run: pytest ./tests
         working-directory: pyvistaqt
+
+  geovista:
+    name: GeoVista
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: git clone https://github.com/bjlittle/geovista.git --single-branch
+      - name: Install PyVista
+        run: pip install -ve . # pyvista
+      - run: sudo apt-get install libgeos-dev
+      - name: Install GeoVista
+        run: pip install -ve . # geovista
+        working-directory: geovista
+      - run: pip install -r requirements/pypi-optional-test.txt
+        working-directory: geovista
+      - run: python -m pytest
```

I'm slightly concerned that the warnings aren't causing this to failed

https://github.com/pyvista/pyvista/actions/runs/5361402806/jobs/9727309100?pr=4568#step:9:1633

Note those warnings are unrelated to PyVista, but we want to fail on warnings to make sure deprecations are handled in GeoVista

---

Good point. This must raise warnings as errors (see [How to capture warnings](https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html)).
```suggestion
      - run: pytest -W error
```